hibernate-rx-core/src/main/java/org/hibernate/rx/engine/impl/RxPersistenceContextAdapter.java

```diff
@@ -44,624 +22,41 @@
-		return delegate.getLoadedCollectionOwnerOrNull( collection );
-	}
-
-	@Override
-	public Serializable getLoadedCollectionOwnerIdOrNull(PersistentCollection collection) {
-		return delegate.getLoadedCollectionOwnerIdOrNull( collection );
-	}
-
-	@Override
-	public void addUninitializedCollection(
-			CollectionPersister persister,
-			PersistentCollection collection, Serializable id) {
-		delegate.addUninitializedCollection( persister, collection, id );
-	}
 
-	@Override
-	public void addUninitializedDetachedCollection(
-			CollectionPersister persister,
-			PersistentCollection collection) {
-		delegate.addUninitializedDetachedCollection( persister, collection );
-	}
-
-	@Override
-	public void addNewCollection(
-			CollectionPersister persister,
-			PersistentCollection collection) throws HibernateException {
-		delegate.addNewCollection( persister, collection );
-	}
-
-	@Override
-	public void addInitializedDetachedCollection(
-			CollectionPersister collectionPersister,
-			PersistentCollection collection) throws HibernateException {
-		delegate.addInitializedDetachedCollection( collectionPersister, collection );
-	}
-
-	@Override
-	public CollectionEntry addInitializedCollection(
-			CollectionPersister persister,
-			PersistentCollection collection, Serializable id) throws HibernateException {
-		return delegate.addInitializedCollection( persister, collection, id );
-	}
-
-	@Override
-	public PersistentCollection getCollection(CollectionKey collectionKey) {
-		return delegate.getCollection( collectionKey );
-	}
-
-	@Override
-	public PersistentCollection getCollectionHolder(Object array) {
-		return delegate.getCollectionHolder( array );
-	}
-
-	@Override
-	public void addCollectionHolder(PersistentCollection holder) {
-		delegate.addCollectionHolder( holder );
-	}
-
-	@Override
-	public PersistentCollection removeCollectionHolder(Object array) {
-		return delegate.removeCollectionHolder( array );
-	}
-
-	@Override
-	public Serializable getSnapshot(PersistentCollection coll) {
-		return delegate.getSnapshot( coll );
-	}
-
-	@Override
-	public CollectionEntry getCollectionEntryOrNull(Object collection) {
-		return delegate.getCollectionEntryOrNull( collection );
-	}
-
-	@Override
-	public Object getProxy(EntityKey key) {
-		return delegate.getProxy( key );
-	}
-
-	@Override
-	public void addProxy(EntityKey key, Object proxy) {
-		delegate.addProxy( key, proxy );
-	}
-
-	@Override
-	public Object removeProxy(EntityKey key) {
-		return delegate.removeProxy( key );
-	}
-
-	@Override
-	@Deprecated
-	public HashSet getNullifiableEntityKeys() {
-		return delegate.getNullifiableEntityKeys();
-	}
-
-	@Override
-	@Deprecated
-	public Map getEntitiesByKey() {
-		return delegate.getEntitiesByKey();
-	}
-
-	@Override
-	public Map.Entry<Object, EntityEntry>[] reentrantSafeEntityEntries() {
-		return delegate.reentrantSafeEntityEntries();
+		final NonLazyCollectionInitializer initializer = new NonLazyCollectionInitializer();
```

Comment:
@gbadner where is this class `NonLazyCollectionInitializer` defined?

Reply:
https://github.com/hibernate/hibernate-rx/blob/e2517f41dca045972cf639f82a1940586e25fcb1/hibernate-rx-core/src/main/java/org/hibernate/rx/engine/impl/RxPersistenceContextAdapter.java#L43

Reply:
Ah, thanks! Didn't see it there.